src/main/java/com/codecool/dungeoncrawl/logic/actors/Actor.java

```diff
@@ -19,6 +23,28 @@ public void move(int dx, int dy) {
         cell = nextCell;
     }
 
+    public void getItem(Object o) {
+        switch (o.getClass().getSimpleName()) {
+            case "Sword":
+                swords.add((Sword) o);
+                break;
+            case "Key":
+                keys.add((Key) o);
+                makeAllMyKeyBlack();
```

Comment:
Why do you repaint all the keys at every new key you pick up, not just the last one?!